### DISCUSSION TITLE
[Enhancement] [tree-v2] el-tree-v2中,希望可以设置高度为“auto”或“100%”，也希望可以提供“width”属性

### DISCUSSION THREAD

**Discussion by fanyawei** [element-plus/element-plus#8684] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-v2`

##  希望改进el-tree-v2的height属性和新增width属性
1、目前高度是通过“height”属性设置，新加的样式会被覆盖，希望能增加string类型，可以设置“auto”、“100%”这类值
2、希望新增“width”属性，可设置的值同上，超出宽度可显示滚动条

### Additional comments

_(empty)_

**Comment by kiddkyd2** [comment]
height这个属性加了之后，滚动的时候ui会错乱。版本2.1.5

**Comment by eric-gitta-moore** [comment]
可以使用el-auto-resizer实现自适应大小 [自动调整大小](https://element-plus.org/zh-CN/component/table-v2.html#%E8%87%AA%E5%8A%A8%E8%B0%83%E6%95%B4%E5%A4%A7%E5%B0%8F)

**Reply by kiddkyd2** [reply]
已测试，可以解决。

**Reply by yaoocheng** [reply]
卧槽。可以这个。`el-auto-resizer`

**Reply by jiangLiu98** [reply]
用这个itemSize

**Reply by zhangke0524** [reply]
有用，赞！使用el-auto-resizer可以实现height跟随父容器的计算高度而自定义，在正确的高度出现滚动条

**Comment by jiangLiu98** [comment]
在组件上使用itemSize可以自定义行高 :itemSize="36"
